circle.yml

```diff
@@ -0,0 +1,6 @@
+machine:
+  node:
+    version: 6.5.0
+dependencies:
+  override:
+    - npm install --dev
```

Comment:
You don't need to `override` because CIRCLE CI would use `npm install` to handle all dependencies.